client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/project-details.component.html

```diff
@@ -1,226 +1,183 @@
+      <label>Forest Region<span class="required">*</span></label>
+      <select id="forestRegionOrgUnitId" formControlName="forestRegionOrgUnitId">
+        <option *ngFor="let item of forestRegionCode" [value]="item.orgUnitId">
+          {{ item.orgUnitName }}
+        </option>
+      </select>
+    </div>
 
-      <div class="form-item">
-        <label>Forest Region</label>
-        <select id="forestRegionOrgUnitId" formControlName="forestRegionOrgUnitId">
-          <option *ngFor="let item of forestRegionCode" [value]="item.orgUnitId">
-            {{ item.orgUnitName }}
-          </option>
-        </select>
-      </div>
+    <div class="form-item" [attr.title]="getSelectedForestDistrictName()">
+      <label> Forest District </label>
+      <select id="forestDistrictOrgUnitId" formControlName="forestDistrictOrgUnitId">
+        <option value="">Select</option>
+        <option *ngFor="let item of forestDistrictCode" [value]="item.orgUnitId">
+          {{ item.orgUnitName }}
+        </option>
+      </select>
+    </div>
 
-      <div class="form-item">
-        <label> Forest District </label>
-        <select id="forestDistrictOrgUnitId" formControlName="forestDistrictOrgUnitId">
-          <option *ngFor="let item of forestDistrictCode" [value]="item.orgUnitId">
-            {{ item.orgUnitName }}
-          </option>
-        </select>
-      </div>
+    <div class="form-item" [attr.title]="getSelectedBcParksRegionName()">
+      <label> BC Parks Region </label>
+      <select id="bcParksRegionOrgUnitId" formControlName="bcParksRegionOrgUnitId">
+        <option value="">Select</option>
+        <option *ngFor="let item of bcParksRegionCode" [value]="item.orgUnitId">
+          {{ item.orgUnitName }}
+        </option>
+      </select>
+    </div>
 
-      <div class="form-item">
-        <label>Primary Objective<span class="required">*</span></label>
-        <select id="fundingStream" formControlName="primaryObjectiveTypeCode">
-          <option *ngFor="let item of objectiveTypeCode" [value]="item.objectiveTypeCode">
-            {{ item.description }}
-          </option>
-        </select>
-      </div>
+    <div class="form-item" [attr.title]="getSelectedBcParksSectionName()">
+      <label>BC Parks Section</label>
+      <select id="bcParksSectionOrgUnitId" formControlName="bcParksSectionOrgUnitId">
+        <option value="">Select</option>
+        <option *ngFor="let item of bcParksSectionCode" [value]="item.orgUnitId">
+          {{ item.orgUnitName }}
+        </option>
+      </select>
+    </div>
 
-      <div class="form-item">
-        <label>Secondary Objective</label>
-        <select id="fundingStream" formControlName="secondaryObjectiveTypeCode">
-          <option *ngFor="let item of objectiveTypeCode" [value]="item.objectiveTypeCode">
-            {{ item.description }}
-          </option>
-        </select>
-      </div>
+    <div class="form-item" [attr.title]="getSelectedFireCentreName()">
```

Comment:
Since we are already using angular material, we should just apply matTooltip here which we can customize something like the postition or styling of the tooltip. It will also align with the tooltip we already apply on other wildfire project.

Reply:
Done and updated screenshots